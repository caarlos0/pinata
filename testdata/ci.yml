name: CI
on: [push]
jobs:
  govulncheck:
    uses: caarlos0/meta/.github/workflows/govulncheck.yml@v0.1.0
  build:
    runs-on: ubuntu-latest
    steps:
      - uses: actions/checkout@v5
      - uses: docker/login-action@v2
      - uses: actions/setup-go@v5
      - uses: github/codeql-action/autobuild@v2
      - uses: github/codeql-action/analyze@192325c86100d080feab897ff886c34abd4c83a3
      - uses: ossf/scorecard-action@v2.4.2
      - uses: github/codeql-action/upload-sarif@192325c86100d080feab897ff886c34abd4c83a3 # v3.30.3
      - uses: codecov/codecov-action@af09b5e394c93991b95a5e7646aeb90c1917f78f # v5.5.1

