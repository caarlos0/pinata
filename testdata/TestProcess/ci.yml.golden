name: CI
on: [push]
jobs:
  govulncheck:
    uses: caarlos0/meta/.github/workflows/govulncheck.yml@c7f17af352dac91fa6c785d06ebac8547f1abdd3 # v0.1.0
  build:
    runs-on: ubuntu-latest
    steps:
      - uses: actions/checkout@08c6903cd8c0fde910a37f88322edcfb5dd907a8 # v5.0.0
      - uses: docker/login-action@465a07811f14bebb1938fbed4728c6a1ff8901fc # v2.2.0
      - uses: actions/setup-go@d35c59abb061a4a6fb18e82ac0862c26744d6ab5 # v5.5.0
      - uses: github/codeql-action/autobuild@b8d3b6e8af63cde30bdc382c0bc28114f4346c88 # v2
      - uses: github/codeql-action/analyze@192325c86100d080feab897ff886c34abd4c83a3
      - uses: ossf/scorecard-action@05b42c624433fc40578a4040d5cf5e36ddca8cde # v2.4.2
      - uses: github/codeql-action/upload-sarif@192325c86100d080feab897ff886c34abd4c83a3 # v3.30.3
      - uses: codecov/codecov-action@af09b5e394c93991b95a5e7646aeb90c1917f78f # v5.5.1

